.github/workflows/build.yml

```
@@ -226,13 +222,24 @@ jobs:
           sudo mv /usr/bin/python3 /usr/bin/python3-default
           sudo ln -s ${{ env.pythonLocation }}/bin/python3 \
           /usr/bin/python3
+      - name: Process user data file
+        id: process-user-data
+        # Process the user data script with the windows admin password
```

More capitalization nitpicking:
```suggestion
        # Process the user data script with the Windows admin password
```

---

resolved with [af4ef4c](https://github.com/cisagov/windows-commando-packer/pull/10/commits/af4ef4cc468b6b9df43d521520e4040d13ba90e2)